src/utils/fetch-twitch-videos.js

```diff
@@ -1,9 +1,13 @@
 import { get } from './request';
+import { TWITCH_HEADERS, TWITCH_VIDEO_URL } from '../constants';
 
 // Fetches videos from twitch api
-const fetchTwitchVideos = async (headers, video_URL, videoLimit = 1) => {
+const fetchTwitchVideos = async (
+    headers = TWITCH_HEADERS,
+    videoURL = TWITCH_VIDEO_URL
```

Comment:
we also can remove passing the headers and video url from everywhere that calls this function, since they are defaults now!